### PR TITLE
updated latest task schema in metadata

### PR DESCRIPTION
**Pull request by shivacharanms**
This PR contains the changes required to update API service metadata section to include the latest task schema version used.

### CODE REVIEW

**Review by jeevan-kamkar — APPROVED** [review]
LGTM

File: svc-api/handle/handlers.go [hunk 1]
```diff
@@ -351,6 +351,7 @@ func GetMetadata(ctx iris.Context) {
 				TopInclude: []models.Include{
 					models.Include{Namespace: "Task"},
 					models.Include{Namespace: "Task.v1_4_2a"},
```
Comment (jeevan-kamkar) [inline svc-api/handle/handlers.go:353]: IMHO proposed version needs to be removed

Comment (shivacharanms) [reply svc-api/handle/handlers.go:353]: Will do thanks

**Review by jeevan-kamkar — APPROVED** [review]
LGTM

**Review by Bharath-KKB — APPROVED** [review]

**Pull request merged**